addons/user-id/userscript.js

```diff
@@ -0,0 +1,11 @@
+export default function ({ addon }) {
+  let usernameHeader = document.querySelector(".header-text").firstElementChild;
+  const userId = Object.assign(document.createElement("span"), {
+    innerText: `#${Scratch.INIT_DATA.PROFILE.model.userId}`,
+    className: "sa-user-id",
+  });
+  userId.style.color = "lightgrey";
+  userId.style.size = 2;
```

Comment:
`"2px"`

Reply:
No, that would make it the size of a rice grain. The "2" is set correctly.

Reply:
Correction: Smaller than the size of a rice grain

Reply:
O right